protobuf/lib/src/protobuf/generated_message.dart

```diff
@@ -95,8 +95,8 @@ abstract class GeneratedMessage {
 
   bool hasRequiredFields() => info_.hasRequiredFields;
 
-  /// Returns [:true:] if all required fields in the message and all embedded
-  /// messages are set, false otherwise.
+  /// Returns `true` if all required fields in the message and all embedded
+  /// messages are set, `false` otherwise.
```

Comment:
Should really have been a getter.
I'd treat it as a property, since it's already named as one, which means also documenting it as a boolean property. 
So another "Whether".